websites/documentation/src/content/configuration/theme.mdx

```diff
@@ -37,6 +37,28 @@ The Docs-Kit theme also receives its plugin configuration through this standard
 
 - `enableCanonicalUrls` (_optional_): indicates that the website should use canonical URLs, pointing to the `docs.commercetools.com` domain. **Default: `true`**
 
+## Available Options for the Site Metadata
```

Comment:
did we really not document this anywhere yet? Thanks!

Reply:
We didn't 😅